src/conwin.f

```diff
@@ -1,66 +1,51 @@
-C>   '>' - GREATER THAN
-C>   '=' - EQUAL 
-C>   '!' - NOT EQUAL
-C>
-C>   EACH CONDITION IN A STRING IS APPLIED TO ONE ELEMENT, AND ALL
-C>   CONDITIONS ARE 'AND'ED TO EVALUATE AN OUTCOME. FOR EXAMPLE, IF THE
-C>   CONDITION STRING IS: "POB<500 TOB>30 TQM<4" THEN THE ONLY LEVELS OF
-C>   DATA READ OR WRITTEN ARE THOSE WITH PRESSURE LT 500 MB, TEMPERATURE
-C>   GT 30 DEG, AND TEMPERATURE QUALITY MARK < 4.
-C>
-C> PROGRAM HISTORY LOG:
-C> 1994-01-06  J. WOOLLEN -- ORIGINAL AUTHOR
-C> 1998-07-08  J. WOOLLEN -- IMPROVED MACHINE PORTABILITY
-C> 1999-11-18  J. WOOLLEN -- THE NUMBER OF BUFR FILES WHICH CAN BE
-C>                           OPENED AT ONE TIME INCREASED FROM 10 TO 32
-C>                           (NECESSARY IN ORDER TO PROCESS MULTIPLE
-C>                           BUFR FILES UNDER THE MPI)
-C> 2003-11-04  S. BENDER  -- ADDED REMARKS/BUFRLIB ROUTINE
-C>                           INTERDEPENDENCIES
-C> 2003-11-04  D. KEYSER  -- MAXJL (MAXIMUM NUMBER OF JUMP/LINK ENTRIES)
-C>                           INCREASED FROM 15000 TO 16000 (WAS IN
-C>                           VERIFICATION VERSION); UNIFIED/PORTABLE FOR
-C>                           WRF; ADDED DOCUMENTATION (INCLUDING
-C>                           HISTORY) 
-C> 2010-04-27  J. WOOLLEN -- CORRECT LOGICAL FLAW AND ADD DOCUMENTATION
-C> 2014-12-10  J. ATOR    -- USE MODULES INSTEAD OF COMMON BLOCKS
+C> @brief Search consecutive subset buffer segments
+C> for an element identified in the user string as a conditional node.
 C>
-C> USAGE:    CALL CONWIN (LUN, INC1, INC2)
-C>   INPUT ARGUMENT LIST:
-C>     LUN      - INTEGER: I/O STREAM INDEX INTO INTERNAL MEMORY ARRAYS
-C>     INC1     - INTEGER: SUBSET BUFFER START INDEX 
-C>     INC2     - INTEGER: SUBSET BUFFER ENDING INDEX
+C> ### Program History Log
+C> Date | Programmer | Comments
+C> -----|------------|---------
+C> 1994-01-06 | J. Woollen | Original author.
+C> 1998-07-08 | J. Woollen | Improved machine portability.
+C> 1999-11-18 | J. Woollen | Increase number of open BUFR FILES.
+C> 2003-11-04 | S. Bender  | Added remarks/bufrlib routine interdependencies.
+C> 2003-11-04 | D. Keyser  | MAXJL increased from 15000 to 16000; unified/portable for WRF.
+C> 2010-04-27 | J. Woollen | correct logical flaw and add documentation.
+C> 2014-12-10 | J. Ator    | use modules instead of common blocks.
 C>
-C>   OUTPUT ARGUMENT LIST:
-C>     INC1     - INTEGER: SUBSET BUFFER START INDEX 
-C>     INC2     - INTEGER: SUBSET BUFFER ENDING INDEX 
+C> @author Woollen @date 1994-01-06
+      
+C> This subroutine searches consecutive subset buffer segments
+C> for an element identified in the user string as a conditional node
+C> (i.e. an element which must meet a condition in order to be read
+C> from or written to a data subset). If a conditional element is
+C> found and it conforms to the condition, then the internal subset
+C> buffer indices of the "window" (see below remarks) are returned to
+C> the caller for processing.
+C> 
```

Comment:
Thanks @edwardhartnett for getting the ball rolling for us on this - much appreciated!

As an FYI for you and for all - whenever Doxygen renders the discussion portion of the docblock, it always cuts off after what it thinks is the first sentence and then renders that as its own separate paragraph.  See [here](https://emc.ncep.noaa.gov/emc/pages/infrastructure/bufrlib_jeff/html_PR253/conwin_8f.html) and scroll down for an example of how it rendered the first sentence of the above paragraph.

So to make the HTML render a little nicer for this routine, how about we re-write it as _"This subroutine searches consecutive subset buffer segments for an element identified in the user string as a conditional node.  A conditional node is an element which must meet a condition in order to be read from or written to a data subset.  If a conditional element..."_